src/presentation-markup.html

```diff
@@ -7691,382 +7665,29 @@ <h5>Table cells that are not divided into alignment groups</h5>
-    unambiguously identified. If no such element is present, renderers should
-    behave as if a zero-width element had been inserted there.</p>
- 
-    <p>For the purposes of alignment, an element X is considered to be to the
-    immediate left of an element Y, and Y to the immediate right of X, whenever
-    X and Y are successive arguments of one (possibly inferred) <code class="element">mrow</code> element,
-    with X coming before Y<span> (in a LTR context; with X coming after Y in a RTL context)</span>. In the case of
-    <code class="element">mfenced</code> elements, MathML applications should evaluate this
-    relation as if the <code class="element">mfenced</code> element had been
-    replaced by the equivalent expanded form involving <code class="element">mrow</code>. Similarly, an <code class="element">maction</code>
-    element should be treated as if it were replaced by its currently selected
-    sub-expression. In all other cases, no relation of <q>to the immediate
-    left or right</q> is defined for two elements X and Y. However, in the
-    case of content elements interspersed in presentation markup, MathML applications
-    should attempt to evaluate this relation in a sensible way. For example, if
-    a renderer maintains an internal presentation structure for rendering
-    content elements, the relation could be evaluated with respect to
-    that. (See <a href="#contm"></a> and <a href="#mixing"></a> for further
-    details about mixing presentation and content markup.)</p>
- 
-    <p>
-     <code class="element">malignmark</code> elements are allowed to occur within
-     the content of token elements, such as <code class="element">mn</code>,
-     <code class="element">mi</code>, or <code class="element">mtext</code>. When this
-     occurs, the character immediately before or after the
-     <code class="element">malignmark</code> element will carry the alignment
-     point; in all other cases, the element to its immediate left or right
-     will carry the alignment point. The rationale for this is that it is
-     sometimes desirable to align on the edges of specific characters
-    within multi-character token elements.</p>
- 
-    <p>If there is more than one <code class="element">malignmark</code> element
+    alignment group, it should only occur within the elements allowed for <code class="element">maligngroup</code>
+    (see <a href="#presm_specify_align_group"></a>).
+    If there is more than one <code class="element">malignmark</code> element
     in an alignment group, all but the first one will be ignored. MathML
     applications may wish to provide a mode in which they will warn about
     this situation, but it is not an error, and should trigger no warnings
     by default. <span>The rationale for this is that</span> it would
     be inconvenient to have to remove all
     unnecessary <code class="element">malignmark</code> elements from
```

Comment:
Why not make it an error?

Reply:
It is more consistent with the "malignmark/maligngroup" can occur anywhere philosophy (ignore if not appropriate) to ignore them.

Reply:
OK I suppose (personally I'd make them invalid, making it more likely editors avoid this, browsers silently ignore invalid markup in any case) But not a battle I want to fight so I'll go with the flow here, marking resolved